src/states/elder/mod.rs

```diff
@@ -1339,24 +1332,27 @@ impl Base for Elder {
         };
 
         if self.chain.is_peer_our_member(&pub_id) {
+            self.parsec_map.block_gossip_recipient(pub_id.name());
```

Comment:
This shall be fine for the fleming branch.
However, once your PR 2050 (lost detection) got merged, may be the `block_gossip_recipient` here is not enough anymore? and shall be at the place of detecting the first failure?

Reply:
Yes, it will be moved to `handle_connection_failure` then.

Reply:
Maybe I can put it there already...

Reply:
yeah, that might be better